app/scripts/lib/experiments/choices/index.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Interface to experiment choices.
+ */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const _ = require('underscore');
+
+  const ExperimentChoices = [
```

Comment:
Just to confirm, all these are experiments can mix together.

If we wanted to create 2 independent experiments then we would to :

```js
 require('./some-name-experiment-chooser'),
 require('./some-name-experiment-1'),
 require('./some-name-experiment-2')
```

Where `some-name-experiment-chooser` would pick between `1` and `2`?

Correct?